cpp/src/barretenberg/honk/composer/composer_helper/standard_honk_composer_helper.cpp

```diff
@@ -111,9 +111,9 @@ std::shared_ptr<bonk::proving_key> StandardHonkComposerHelper<CircuitConstructor
     StandardHonkComposerHelper::compute_proving_key_base(circuit_constructor, plonk::ComposerType::STANDARD_HONK);
 
     // Compute sigma polynomials (we should update that late)
-    compute_standard_honk_sigma_permutations<CircuitConstructor::program_width>(circuit_constructor,
```

Comment:
Since in this PR there is now an array `wires`, it made sense to finally change to the more descriptive name `num_wires`.